contentgrid-appserver-query-engine-impl-jooq/src/main/java/com/contentgrid/appserver/query/engine/jooq/JOOQTableCreator.java

```diff
@@ -6,28 +6,82 @@
 import com.contentgrid.appserver.application.model.attributes.CompositeAttribute;
 import com.contentgrid.appserver.application.model.attributes.SimpleAttribute;
+import com.contentgrid.appserver.application.model.exceptions.InvalidArgumentModelException;
+import com.contentgrid.appserver.application.model.searchfilters.AttributeSearchFilter;
 import com.contentgrid.appserver.query.engine.api.TableCreator;
 import com.contentgrid.appserver.query.engine.api.exception.InvalidSqlException;
 import com.contentgrid.appserver.query.engine.jooq.resolver.DSLContextResolver;
 import com.contentgrid.appserver.query.engine.jooq.strategy.JOOQRelationStrategyFactory;
+import lombok.NonNull;
 import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.jooq.Allow;
 import org.jooq.CreateTableElementListStep;
 import org.jooq.DSLContext;
 import org.jooq.impl.DSL;
+import org.springframework.core.io.ClassPathResource;
 import org.springframework.jdbc.BadSqlGrammarException;
 import org.springframework.transaction.annotation.Transactional;
 
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.*;
+
+import static java.util.Locale.ENGLISH;
+import static java.util.Map.entry;
+
+@Slf4j
 @RequiredArgsConstructor
 @Transactional
 public class JOOQTableCreator implements TableCreator {
 
+    static final @NonNull Set<@NonNull Locale> SUPPORTED_LOCALES = Set.of(
+            Locale.of("ar"), // Arabic
+            Locale.of("hy"), // Armenian
+            Locale.of("eu"), // Basque
+            Locale.of("ca"), // Catalan
+            Locale.of("da"), // Danish
+            Locale.of("nl"), // Dutch
+            Locale.of("en"), // English
+            Locale.of("fi"), // Finnish
+            Locale.of("fr"), // French
+            Locale.of("de"), // German
+            Locale.of("el"), // Greek
+            Locale.of("hi"), // Hindi
+            Locale.of("hu"), // Hungarian
+            Locale.of("id"), // Indonesian
+            Locale.of("ga"), // Irish
+            Locale.of("it"), // Italian
+            Locale.of("lt"), // Lithuanian
+            Locale.of("ne"), // Nepali
+            Locale.of("no"), // Norwegian
+            Locale.of("pt"), // Portuguese
+            Locale.of("ro"), // Romanian
+            Locale.of("ru"), // Russian
+            Locale.of("sr"), // Serbian
+            Locale.of("es"), // Spanish
+            Locale.of("sv"), // Swedish
+            Locale.of("ta"), // Tamil
+            Locale.of("tr"), // Turkish
+            Locale.of("yi")  // Yiddish
+    );
+    private static final @NonNull String ftsIndexPreparedStatement;
+
+    static {
+        try (InputStream inputStream = new ClassPathResource("com/contentgrid/appserver/query/engine/jooq/sql/statements/create_fts_index.sql").getInputStream()) {
+            ftsIndexPreparedStatement = new String(inputStream.readAllBytes());
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
Why does this need to be in a file that is loaded, instead of just an inline string?